pkgs/development/python-modules/sh/default.nix

```diff
@@ -9,10 +9,15 @@ buildPythonPackage rec {
     sha256 = "1z2hx357xp3v4cv44xmqp7lli3frndqpyfmpbxf7n76h7s1zaaxm";
   };
 
-  checkInputs = [ coverage ];
+  postPatch = ''
+    sed -i 's#/usr/bin/env python#${python.interpreter}#' test.py
```

Comment:
`patchShebangs` does not work because the shebang is embedded in the test code, not at the beginning of the file. See: https://github.com/amoffat/sh/blob/master/test.py#L2772